deploy/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Deploy to Kubernetes'
+description: 'Deploy an application to a Kubernetes cluster.'
+
+inputs:
+  deployment-file:
+    description: '"test" or "prod"'
```

Comment:
I would add a working directory here. This gives us way more flexibility.

https://github.com/aboutbits/github-actions-node/blob/main/setup-and-install/action.yml#L4-L7

Reply:
I would like to take a look at working directories at a later point. This is related to monorepos where the actual project might not reside in the root of the repository.
Github actions already provides an option for `working-directory` and I don't want to mix them up.
Also I would postpone this to a later time because there are more pressing issues at the moment.